pkg/alert/prometheusrule.go

```diff
@@ -117,6 +119,12 @@ func (r *multiWindowMultiBurnAlertRule) render(url string, percent string, names
 		" and " +
 		sufficientProbes(r.longWindow, labelSelector)
 
+	if namespacedName.Name == "console" {
+		defaultConsoleURL := strings.TrimSuffix(url, "/health")
```

Comment:
How do we normally determine the console URL is default or custom? Just checking if it's in the `console` namespace?

Reply:
the `url` here comes from the `routeMonitor.Status.RouteURL` which was set to the default and I did not see it  would change

Reply:
Sorry, I am still not quite clear how's the `url` comes from the `routeMonitor.Status.RouteURL` from the function calls. Do you mean it's derived from this struct?
```
// RouteMonitorStatus defines the observed state of RouteMonitor
type RouteMonitorStatus struct {
	// RouteURL is the url extracted from the Route resource
	RouteURL          string         `json:"routeURL,omitempty"`
	ServiceMonitorRef NamespacedName `json:"serviceMonitorRef,omitempty"`
	PrometheusRuleRef NamespacedName `json:"prometheusRuleRef,omitempty"`
	ErrorStatus       string         `json:"errorStatus,omitempty"`
}
```
But how's `RouteURL` above is determiend as the default URL? Is that explicitly defined something like `RouteURL: "https://console-openshift-console.apps.example.com/health`?

Reply:
The route fetched by the other function `GetRoute()` in routemonitor controller by reading the the route with name `console` in openshift-console ns, and append the suffix ("/health" here)